File: TimeTrackerPlanerWeb/TimeTrackerPlanerWeb/Models/TasksPlanned.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace TimeTrackerPlanerWeb.Models
{
    public class TasksPlanned
    {
        [Key]
        public int planid { get; set; }
        public int taskid { get; set; }

        public virtual TasksDoing TasksDoing { get; set; }
    }
}

File: ttpLaravel/app/TasksPlanned.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;

class tasksplanned extends Model
{
    protected $table = 'TasksPlanned';

    public function tasksdoing()
    {
        return $this->hasMany(tasksdoing::class, 'planid', 'planid');
    }

    public function tasknames()
    {
        return $this->belongsTo(tasknames::class, 'taskid', 'taskid');
    }
}
